terraform/modules/admin-lb/main.tf

```diff
@@ -30,13 +30,37 @@ resource "kubernetes_manifest" "admin-server-cert" {
   }
 }
 
+resource "null_resource" "wait_for_server_cert_creation" {
+  provisioner "local-exec" {
+    command = <<-EOT
+    COUNTER=0
+    MAX_TRIES=50
+    while ! kubectl describe managedcertificate ${var.admin_server_cert_id} --namespace=${var.deployment_namespace} | grep -i "certificate name" && [ $COUNTER -lt $MAX_TRIES ]
+    do
+      sleep 15
```

Comment:
Set the sleep time in an environment variable as well, so you've all the parameters in a single place.

Reply:
done